### DISCUSSION TITLE
Add Random Agent

### DISCUSSION THREAD

**Discussion by R43d-SEC** [projectdiscovery/nuclei#4559] [Ideas]
Please Add Random Agent

**Comment by geeknik** [comment]
Random user agents has been the nuclei default for many, many versions now.

**Comment by R43d-SEC** [comment]
whats this argument key ? i don't see in help

**Comment by geeknik** [comment]
it is on by default, there is no command line argument for it. it was removed a while ago. 👍🏻

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
